packages/okta-react-native/index.js

```diff
@@ -0,0 +1,86 @@
+import { NativeModules, Platform } from 'react-native';
+import { assertIssuer, assertClientId, assertRedirectUri } from '@okta/configuration-validation';
+import jwt from 'jwt-lite';
+
+export const createConfig = async ({
+  clientId,
+  redirectUri, 
+  endSessionRedirectUri, 
+	discoveryUri, 
+	scopes
+}) => {
+
+  assertIssuer(discoveryUri);
+  assertClientId(clientId);
+  assertRedirectUri(redirectUri);
+  assertRedirectUri(endSessionRedirectUri);
+
+	if (Platform.OS === 'ios') {
+		scopes = scopes.join(' ');
+	}
+    
+  return NativeModules.OktaSdkBridge.createConfig(
+    clientId,
+    redirectUri,
+    endSessionRedirectUri,
+    discoveryUri,
+    scopes
+  );
+} 
+
+export const signIn = async () => {
```

Comment:
nit style: check for the space in `async ()` declaration, sometimes there is no space between async and ()

Reply:
Thanks for checking this, I deleted all the space to keep it consistent